scout/adapter/mongo/hpo.py

```diff
@@ -105,85 +105,3 @@ def hpo_terms(self, query=None, hpo_term=None, text=None, limit=None):
         )
 
         return res
-
-    def disease_term(self, disease_identifier):
```

Comment:
I didn't delete these, I just moved to a new handler (Diagnosis handler), since these are diagnosis related (OMIM) and not strictly phenotypes (HPO)